features/dd-sdk-android-webview/src/main/kotlin/com/datadog/android/webview/WebViewTracking.kt

```diff
@@ -121,12 +125,25 @@ object WebViewTracking {
         if (webViewLogsFeature == null && webViewRumFeature == null) {
             return NoOpWebViewEventConsumer()
         } else {
+            // it is very important that the timestamp offset provider is shared between the
+            // different consumers, otherwise we might end up with different offsets for the replay
+            // and rum browser events for the same view id.
+            val timestampOffsetProvider = TimestampOffsetProvider(sdkCore.internalLogger)
```

Comment:
we should also probably apply it to the `WebViewLogEventConsumer`? we do time correction there.

Reply:
yes we do but we do not have the same constraints and no `viewId` involved.

Reply:
> and no viewId involved.
Isn't there a bundled RUM info in the logs in the browser SDK ?

Reply:
I need to check this actually.